spec/rental_spec.rb

```diff
@@ -0,0 +1,28 @@
+require_relative '../rental'
+require_relative '../person'
+require_relative '../book'
+
+describe Rental do
+  describe '#initialize' do
+    it 'should set the date, book, and person' do
+      book = { 'title' => 'The Great Gatsby', 'author' => 'F. Scott Fitzgerald', 'rentals' => [] }
+      person = { 'name' => 'John Doe', 'age' => 25, 'parent_permission' => true, 'rentals' => [] }
+      date = '2022-01-01'
+      rental = Rental.new(date, person, book)
+
+      expect(rental.date).to eq date
+      expect(rental.book).to eq book
+      expect(rental.person).to eq person
+    end
+
+    it 'should add the rental to the book and person rentals' do
+      book = { 'title' => 'The Great Gatsby', 'author' => 'F. Scott Fitzgerald', 'rentals' => [] }
+      person = { 'name' => 'John Doe', 'age' => 25, 'parent_permission' => true, 'rentals' => [] }
+      date = '2022-01-01'
+      rental = Rental.new(date, person, book)
+
+      expect(book['rentals'].last).to eq rental
+      expect(person['rentals'].last).to eq rental
+    end
+  end
```

Comment:
- [ ] @team great job on your tests, however, you forgot to add a test for the  `to_h` method, its a project requirement to have unit tests for all methods in a class.